testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js

```diff
@@ -36,25 +43,40 @@ const moduleConfig = {
     }
 };
 
+// How to view a generated ExcelJS workbook in Excel:
+// 1. Add '<script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/1.3.8/FileSaver.min.js"></script>' to 'testing\runner\Views\Main\RunSuite.cshtml'
+// 2. Call 'then' function of the exportDataGrid' function result and save workbook to file:
+//    .then(() => {
+//        this.worksheet.workbook.xlsx.writeBuffer().then(function(buffer) {
+//            saveAs(new Blob([buffer], { type: "application/octet-stream" }), "DataGrid.xlsx");
+//        });
+//    })
+// 3. Select a file in the shown 'SaveAs' dialog and open the saved file in Excel
+
 QUnit.module("API", moduleConfig, () => {
     [undefined, { row: 1, column: 1 }, { row: 2, column: 3 }].forEach((topLeftCell) => {
         let topLeft = (topLeftCell ? topLeftCell : { row: 1, column: 1 });
         let topLeftCellOption = `, topLeftCell: ${JSON.stringify(topLeftCell)}`;
 
-        const getConfig = (dataGrid, expectedCustomizeCellArgs) => ({
-            component: dataGrid,
-            worksheet: this.worksheet,
-            topLeftCell: topLeftCell,
-            customizeCell: (eventArgs) => {
-                if(typeUtils.isDefined(expectedCustomizeCellArgs)) {
-                    helper.checkCustomizeCell(eventArgs, expectedCustomizeCellArgs, this.customizeCellCallCount++);
-                }
-            }
-        });
-
         [true, false].forEach((excelFilterEnabled) => {
             let options = topLeftCellOption + `, excelFilterEnabled: ${excelFilterEnabled}`;
-            const getDataGridConfig = (dataGrid, expectedCustomizeCellArgs) => $.extend(getConfig(dataGrid, expectedCustomizeCellArgs), { excelFilterEnabled: excelFilterEnabled });
+            const getDataGridConfig = (dataGrid, expectedCustomizeCellArgs, exportColumnWidth) => {
+                const result = {
+                    component: dataGrid,
+                    worksheet: this.worksheet,
+                    topLeftCell: topLeftCell,
+                    customizeCell: (eventArgs) => {
+                        if(typeUtils.isDefined(expectedCustomizeCellArgs)) {
+                            helper.checkCustomizeCell(eventArgs, expectedCustomizeCellArgs, this.customizeCellCallCount++);
+                        }
+                    },
+                    excelFilterEnabled: excelFilterEnabled,
+                };
+                if(exportColumnWidth !== undefined) {
```

Comment:
```suggestion
                if(typeUtils.isDefined(exportColumnWidth)) {
```

Reply:
In this case it's better to change arguments to "const getDataGridConfig = (options)" and use the passed values.
I have tried this approach but it requires a lot of changes and makes this commit too large.
Currently, there are explicit arguments instead of such object and this is why I used 'undefined': it is assigned to an argument if the calling code doesn't specify its value.
'null' value means a set of another use cases that should not be covered by this code.